### PR TITLE
[New port] Add cpp-TimSort 2.1.0

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Adds new port for version 2.1.0 of [cpp-TimSort](https://github.com/timsort/cpp-TimSort).

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/cpp-timsort/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/gfx PACKAGE_NAME gfx-timsort)
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
+
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/cpp-timsort RENAME copyright)
```
Comment (JonLiu1993) [inline ports/cpp-timsort/portfile.cmake:22]:
```suggestion
file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

File: ports/cpp-timsort/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "cpp-timsort",
+  "version-string": "2.1.0",
```
Comment (JonLiu1993) [inline ports/cpp-timsort/vcpkg.json:3]:
```suggestion
  "version": "2.1.0",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

File: ports/cpp-timsort/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/gfx PACKAGE_NAME gfx-timsort)
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
+
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (dg0yt) [inline ports/cpp-timsort/portfile.cmake:22]: Double quotes missing here.

Comment (JonLiu1993) [reply ports/cpp-timsort/portfile.cmake:22]: @dg0yt ,Thanks for your reminder, I will add and resubmit

**Comment by BillyONeal** [comment]
Thanks for the new port!

**Pull request merged**